src/test/resources/local-pipeline/run_local_pipeline

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+echo "Checking args"
+if [ $# -ne 5 ];
+then
+	echo "Execution using 5 (five) arguments:"
+	echo "arg1 - usgs username."
+	echo "arg2 - usgs password."
+	echo "arg3 - image dataset in format landsat_X, where X (landsat number)."
+	echo "arg4 - image region in format PPPRRR, where P (path) and R (row)."
+	echo "arg5 - image date in format YYYY-MM-DD, where Y (year), M (month) and D (day)."
+	exit 1
```

Comment:
make it more standard. e.g

echo "Usage: $0 usgs_username usgs_password [landsat_5 | landsat_7 | landsat_8] region_pathrow image_date"

Reply:
add an exempla, omitting user and password in the script comments